Tests/LibGL/TestRender.cpp

```diff
@@ -317,3 +318,48 @@ TEST_CASE(0011_tex_env_combine_with_constant_color)
     context->present();
     expect_bitmap_equals_reference(context->frontbuffer(), "0011_tex_env_combine_with_constant_color"sv);
 }
+
+TEST_CASE(0012_blend_equations)
+{
+    auto context = create_testing_context(64, 64, Gfx::BitmapFormat::BGRA8888);
+
+    // Clear with alpha 0 so we get a transparent color buffer
+    glClearColor(0, 0, 0, 0);
+    glClear(GL_COLOR_BUFFER_BIT);
+
+    glColor4f(.8f, .2f, .3f, .7f);
+    glRecti(-1, -1, 1, 1);
+
+    glColor4f(.3f, .1f, .8f, .5f);
+    glEnable(GL_BLEND);
+    glBlendFunc(GL_ONE, GL_ONE);
+
+    static constexpr Array<GLenum, 5> blend_modes = {
+        GL_FUNC_ADD,
+        GL_FUNC_SUBTRACT,
+        GL_FUNC_REVERSE_SUBTRACT,
+        GL_MIN,
+        GL_MAX,
+    };
+    auto constexpr grid_size = blend_modes.size();
+    auto constexpr cell_size = 2.f / grid_size;
+    for (size_t x = 0; x < grid_size; ++x) {
+        for (size_t y = 0; y < grid_size; ++y) {
+            auto rgb_mode = blend_modes[x];
+            auto alpha_mode = blend_modes[y];
+
+            glBlendEquationSeparate(rgb_mode, alpha_mode);
```

Comment:
Want to add a glGet() test right after while here? (Up to you)

Reply:
Good idea, I've also added two assertions for the initial state.